ci/do_ci.sh

```diff
@@ -55,6 +68,20 @@ elif [[ "$1" == "cmake.c++20.test" ]]; then
   make
   make test
   exit 0
+elif [[ "$1" == "cmake.c++20.stl.test" ]]; then
+  install_prometheus_cpp_client
+  cd "${BUILD_DIR}"
+  rm -rf *
+  cmake -DCMAKE_BUILD_TYPE=Debug  \
+        -DWITH_METRICS_PREVIEW=ON \
+        -DWITH_LOGS_PREVIEW=ON \
+        -DCMAKE_CXX_FLAGS="-Werror" \
+        -DCMAKE_CXX_STANDARD=20 \
```

Comment:
You don't have to manually specify `CMAKE_CXX_STANDARD=20`. I fixed it so that it gets properly asked-for and detected in #860  . You can just omit it because `WITH_STL=ON` is going to set it either to 17 or 20, depending on CMake version. Older CMake versions did not know about 20.

Reply:
nice, will do tha.